src/objects/button/button.stories.mdx

```diff
@@ -35,3 +37,15 @@ Some buttons may not require as much emphasis as others. The `o-button--secondar
     })}
   </Story>
 </Preview>
+
+All visual styles are optimized for [our dark theme](/?path=/docs/themes--dark). We avoid simply inverting the visual appearance so buttons will appear distinct from white content containers or text inputs.
```

Comment:
>We avoid simply inverting the visual appearance so buttons will appear distinct from white content containers or text inputs.

For some reason my brain struggles to tell if "buttons will appear distinct from white content containers or text inputs" happens when we do or don't invert.

Maybe "Instead of simply inverting their appearance we have different dark theme styles so our buttons appear distinct from white content containers or text inputs."

Might just be me though.

Reply:
I re-wrote this paragraph, it was very clunky before.